python/ml4ir/applications/ranking/model/ranking_model.py

```diff
@@ -1,6 +1,7 @@
 import tensorflow as tf
 from tensorflow import data
 import os
+import sys
```

Comment:
minor: but the order of the imports here should follow core python packages, pip-installed and ml4ir.
So, `os`, `sys` should go to the top.